tests/test_mongodb_full_table.py

```diff
@@ -275,13 +282,14 @@ def test_run(self):
             self.assertNotEqual(first_versions[tap_stream_id], second_versions[tap_stream_id])
 
             # version which is larger than the previous target version
-            self.assertTrue(second_versions[tap_stream_id]>first_versions[tap_stream_id])
+            self.assertTrue(second_versions[tap_stream_id] > first_versions[tap_stream_id])
 
             # verify that menagerie state does include the version which matches the target version
             self.assertEqual(records_by_stream[self.tap_stream_id_to_stream()[tap_stream_id]]['table_version'], second_versions[tap_stream_id])
 
+            # TODO: check with kyle on the below two assertions, they are identical to the above two assertions
```

Comment:
Yeah these are no necessary if they are deplicates. Also we should remove the assertTrue and use assertGreater

Reply:
done